src/sagemaker/model.py

```diff
@@ -36,6 +37,8 @@
     ["mxnet", "tensorflow", "keras", "pytorch", "onnx", "xgboost", "tflite"]
 )
 
+NEO_PYTORCH_DEFAULT_VERSION = "1.6"
```

Comment:
a default framework version was removed from the various frameworks in sdk v2 (july 2020) to promote explicitness and production stability